### PR TITLE
Document usage of Pytest with Tox as a FAQ entry

### LINKED ISSUE #3187: Document that temp dir tweaks should be applied when running Pytest in parallel with Tox
## What's the problem this feature will solve?

Parallel runs of Pytest are supported only when the base temporary directory is unique for each concurrent run. For Tox 3, this is documented by way of an example here: <https://tox.wiki/en/3.27.0/example/pytest.html#extended-example-change-dir-before-test-and-use-per-virtualenv-tempdir>

## Describe the solution you'd like

I appreciate that this really is a feature of Pytest and not Tox. However, since Tox provides a unique temporary directory per virtualenv out of the box, it may be helpful to document how this can be used to run Pytest with Tox's `--parallel` option.

I don't know **where** within the Tox 4 documentation pages this should be included though...

## Alternative Solutions

Do nothing: keep the documentation as is.

## Additional context

None.

### PR DESCRIPTION
**Pull request by faph**
Fixes #3187 

- [x] ran the linter to address style issues (`tox -e fix`)
- [x] wrote descriptive pull request text
- [x] ensured there are test(s) validating the fix
- [x] added news fragment in `docs/changelog` folder
- [x] updated/extended the documentation

### CODE REVIEW

**Comment by faph** [comment]
@gaborbernat Let me know what you think. This is loosely based on the Tox 3 docs with some additional detail in places.

File: docs/faq.rst [hunk 1]
```diff
@@ -366,3 +366,48 @@ If you need to test against e.g. Python 2.7, 3.5 or 3.6, you need to add the fol
...
+    commands = pytest --basetemp="{env_tmp_dir}"
+
+Setting the ``pytest`` ``--basetemp`` argument also causes all temporary ``pytest`` files to be deleted immediately
+after the tests are completed. To restore the default ``pytest`` behavior to retain temporary files for the most recent
+``pytest`` invocations, the system's temporary directory location could be configured like this instead:
+
+.. code-block:: ini
+
+    [tox]
+    setenv =
```
Comment (gaborbernat) [inline docs/faq.rst:409]: setenv -> set_env

**Review by gaborbernat — CHANGES_REQUESTED** [review]
Otherwise looks good.

File: docs/faq.rst (hunk 1 above)
Comment (faph) [reply docs/faq.rst]: Done

**Comment by faph** [comment]
Thanks!

**Pull request merged**